lib/matplotlib/widgets.py

```diff
@@ -1653,7 +1653,7 @@ def clear(self, event):
     def onmove(self, event):
         if self.ignore(event):
             return
-        if event.inaxes is None:
+        if not event.inaxes in self.axes:
```

Comment:
```suggestion
        if event.inaxes not in self.axes:
```
This should make the flake checker happy

Reply:
indeed. thanks!